docker/setup.sh

```diff
@@ -0,0 +1,178 @@
+#!/bin/sh
```

Comment:
This is our user-facing installer script, and shouldn't be used for our docker infrastructure